terraform/modules/data-share-service/cloudwatch.tf

```diff
@@ -41,22 +41,40 @@ module "metrics_dashboard" {
       period = local.metric_period,
       stat   = "Sum",
       metrics = [
-        { name = "API_CALLS.PublishEvent.count", dimensions = { success = true }, attributes = { label = "PublishEvent" } },
+        { name       = "API_CALLS.PublishEvent.count", dimensions = { success = true },
+          attributes = { label = "PublishEvent" }
+        },
         { name = "API_CALLS.GetEvent.count", dimensions = { success = true }, attributes = { label = "GetEvent" } },
         { name = "API_CALLS.GetEvents.count", dimensions = { success = true }, attributes = { label = "GetEvents" } },
-        { name = "API_CALLS.GetEventsStatus.count", dimensions = { success = true }, attributes = { label = "GetEventsStatus" } },
-        { name = "API_CALLS.DeleteEvent.count", dimensions = { success = true }, attributes = { label = "DeleteEvent" } },
+        { name       = "API_CALLS.GetEventsStatus.count", dimensions = { success = true },
+          attributes = { label = "GetEventsStatus" }
+        },
+        { name       = "API_CALLS.DeleteEvent.count", dimensions = { success = true },
+          attributes = { label = "DeleteEvent" }
+        },
       ]
     },
     {
       title  = "Data ingest calls",
       period = local.metric_period,
       stat   = "Sum",
       metrics = [
-        { name = "API_CALLS.PublishEvent.count", dimensions = { success = true }, attributes = { label = "PublishEvent" } },
+        { name       = "API_CALLS.PublishEvent.count", dimensions = { success = true },
+          attributes = { label = "PublishEvent" }
+        },
         { name = "API_CALLS.CallsToLev.count", attributes = { label = "CallsToLev" } },
         { name = "API_RESPONSES.ResponsesFromLev.count", attributes = { label = "ResponsesFromLev" } },
       ]
     },
+    {
+      title  = "Data processing times",
+      period = local.metric_period,
+      stat   = "p99",
+      metrics = [
+        { name       = "DATA_PROCESSING.TimeFromCreationToDeletion",
+          attributes = { label = "99th percentile time from data creation to deletion" }
```

Comment:
I think the label here will be on the metric, so maybe do `Data creation to deletion time` and then update the title to `Data processing times (p99)`